core/bin/snapshots_creator/README.md

```diff
@@ -1,24 +1,49 @@
-to a certain L1 Batch.
+Snapshot creator is a command line tool for creating an app-level snapshot of the node storage at a certain L1 batch. A
+snapshot does not contain full transaction history, but rather [a minimal subset of information](#snapshots-format)
+needed to bootstrap an external node. A snapshot is always made at an L1 batch boundary.
 
-Snapshots do not contain full transactions history, but rather a minimal subset of information needed to bootstrap EN
-node.
+Compared to Postgres dumps, app-level snapshots are much more compact; as of Feb 2024, the mainnet snapshot is approx.
+13 GB (gzipped).
 
-Usage (local development):\
-First run `zk env dev` \
-then the creator can be run using:  
-`zk run snapshots_creator`
-
-Snapshot contents can be stored based on blob_store config either in local filesystem or GS.
+> **Warning.** Snapshot creator is in the early stage of development; expect bugs and incomplete / outdated docs. Use at
+> your own risk.
 
 ## Snapshots format
 
-Each snapshot consists of three types of objects (see
-[snapshots.rs](https://github.com/matter-labs/zksync-era/blob/main/core/lib/types/src/snapshots.rs)) : header, storage
-logs chunks and factory deps:
+Each snapshot consists of three types of data (see [`snapshots.rs`] for exact definitions):
+
+- **Header:** Includes basic information, such as the miniblock / L1 batch of the snapshot, miniblock / L1 batch
+  timestamps, miniblock hash and L1 batch root hash. Returned by the methods in the `snapshots` namespace of the
+  JSON-RPC API of the main node.
+- **Storage log chunks:** Latest values for all VM storage slots ever written to at the time the snapshot is made.
+  Besides key–value pairs, each storage log record also contains the L1 batch number of its initial write and its
+  enumeration index; both are used to restore the contents of the `initial_writes` table. Chunking storage logs is
+  motivated by their parallel generation; each chunk corresponds to a distinct non-overlapping range of hashed storage
+  keys. (This should be considered an implementation detail for the purposes of snapshot recovery; recovery must not
+  rely on any particular key distribution among chunks.) Stored as gzipped Protobuf messages in an [object store]; each
+  chunk is a separate object.
+- **Factory dependencies:** All bytecodes deployed on L2 at the time the snapshot is made. Stored as a single gzipped
+  Protobuf message in an object store.
+
+## Local testing
+
+Usage for local development (assuming the development environment [has been set up](../../../docs/guides/setup-dev.md)):
+
+1. Run `zk env dev`
+2. Generate storage logs, e.g. by running a load test with a command like
+   `ACCOUNTS_AMOUNT=40 DURATION_SEC=60 zk run loadtest`.
+3. Run the creator using `zk run snapshots-creator`.
+
+Snapshot contents can be stored based on the object store configuration either in the local filesystem or Google Cloud
```

Comment:
I'd revert the order, first tell where to find them by default and only then mention that it can be configured to be stored in GCS. We could also mention, what are the paths to snapshots bucket on testnet and mainnet

Reply:
I think it would also be useful to add a curl snippet to fetch info about all snapshots from mainnet node, but probably in the first section of this document